### DISCUSSION TITLE
Command arguments

### DISCUSSION THREAD

**Discussion by paulkczw** [Minestom/Minestom#1839] [Help] [answered]
Hi, am I doing something wrong? Because "/test cpu" runs the default executor

Code:
https://hastebin.com/share/uladuhubuh.kotlin

**Comment by Redstonneur1256** [comment, marked as answer]
For me, moving the syntax with only the cpu literal before the other two seemed to fix the problem.

**Reply by paulkczw** [reply]
Yes, that's the way it works for me too. Seems to be an issue, right?
